askama_shared/src/helpers/mod.rs

```diff
@@ -1,6 +1,6 @@
-use std::iter::Enumerate;
-use std::iter::Peekable;
+use std::iter::{Enumerate, Peekable};
 
+#[allow(missing_debug_implementations)]
```

Comment:
I don't consider the `askama_shared` interface public even though it technically is, so I don't think following the advice about applying `Debug` ~everywhere is useful here.